### PR TITLE
metiq: Calculate correlation coefficient for max

### PR DESCRIPTION
**Pull request by JohanBlome**
Calculate an int val between 0-100 where 100 is max correlation.

### CODE REVIEW

File: src/audio_analyze.py [hunk 1]
```diff
@@ -110,7 +110,7 @@ def audio_analyze_wav(infile, **kwargs):
     index_list, correlation = get_correlation_indices(inaud, needle)
     # add a samplerate-based timestamp
     audio_results = [
-        (index, index / samplerate, correlation[index]) for index in index_list
+        (index, index / samplerate, int(np.corrcoef(inaud[index: index + len(needle)], needle)[1, 0] * 100)) for index in index_list
```
Comment (chemag) [inline src/audio_analyze.py:113]: Happy to change the correlation code, but then we do not need the code above calculating the `correlation` variable. Please clean

Comment (JohanBlome) [reply src/audio_analyze.py:113]: Sorry. Please clarify. Do we want to have the correlation coefficient or the correlation values as output?
Without measuring I think calculating correlation coefficients for the entire vector is much more expensive which is why I put it here and only for the actual used values.

Comment (chemag) [reply src/audio_analyze.py:113]: Before this patch, we are using the values in the `correlation` array, calulcated using `get_correlation_indices()`) to decide where are the original beeps. 

The patch is instead using `np.corrcoef()`. 

It should be one or the other, right?

Comment (JohanBlome) [reply src/audio_analyze.py:113]: Not really. The corrcoeff is more expensive and the way we have the system setup we need to calculate the correlation value for every sample.
This is much less expensive and seems to work fine.

Comment (JohanBlome) [reply src/audio_analyze.py:113]: This is just to inform the user of the confidence level.

**Pull request merged**